test/test-views/tests.hpp

```diff
@@ -0,0 +1,23 @@
+/**
+    Copyright 2020 Red Anchor Trading Co. Ltd.
+
+    Distributed under the Boost Software License, Version 1.0.
+    See <http://www.boost.org/LICENSE_1_0.txt>
+ */
+
+
+#pragma once
+
+
+#include <fost/urlhandler>
+
+
+namespace test {
+
+
+    namespace view {
+
+
+        extern const fostlib::urlhandler::view &chain_request;
```

Comment:
I don't think we need this file. We've deprecated all of these variables and removed them all elsewhere